crates/zed/src/languages/erlang/config.toml

```diff
@@ -0,0 +1,22 @@
+name = "Erlang"
+# TODO: support parsing rebar.config files
+# # https://github.com/WhatsApp/tree-sitter-erlang/issues/3
+path_suffixes = ["erl", "hrl", "app.src", "escript", "xrl", "yrl", "Emakefile", "rebar.config"]
+line_comments = ["% ", "%% ", "%%% "]
+autoclose_before = ";:.,=}])>"
+brackets = [
+    { start = "{", end = "}", close = true, newline = true },
+    { start = "[", end = "]", close = true, newline = true },
+    { start = "(", end = ")", close = true, newline = true },
+    { start = "<<", end = ">>", close = true, newline = false, not_in = ["string"] },
+    { start = "\"", end = "\"", close = true, newline = false, not_in = ["string"] },
+    { start = "'", end = "'", close = true, newline = false, not_in = ["string", "comment"] },
+]
+# Indent if a line ends brackets, "->" or most keywords. Also if prefixed
+# with "||". This should work with most formatting models.
+increase_indent_pattern = "^*([{(\\[]]|\\->|after|begin|case|catch|fun|if|of|try|when|maybe|else|(\\|\\|.*))\\s*$"
```

Comment:
Ideally, we don't do this indentation in comments but Zed does not support `look-around` in regexes, there is a [fancy-regex](https://crates.io/crates/fancy-regex) crate that does which will be a nice thing to have in Zed